pom.xml

```diff
@@ -182,32 +180,11 @@
         <artifactId>maven-settings</artifactId>
         <version>${mavenVersion}</version>
       </dependency>
-      <!-- doxia -->
+      <!-- shared -->
       <dependency>
-        <groupId>org.apache.maven.doxia</groupId>
-        <artifactId>doxia-sink-api</artifactId>
-        <version>${doxiaVersion}</version>
-        <exclusions>
-          <exclusion>
-            <groupId>org.codehaus.plexus</groupId>
-            <artifactId>plexus-container-default</artifactId>
```

Comment:
I we have it on dependency list will generate warning in 3.9.2

Reply:
Which exactly? This has been deleted. You mean maven-reporting-api?

Reply:
This problem will be solved when we move to Doxia 2.0.0. For now, we need to exclude otherwise things start to fail. This is OK for me for now.

Reply:
ok